### ISSUE TITLE
"Next" Button on Name page doesn't work

### ISSUE THREAD

**Issue by im-skylar** [RetroMusicPlayer/Paisa#433]
When I try to press next after entering my name nothing happens.

I just updated from the play store version to 6.0.0 from here on GitHub and now the Next Button on the page with "Hi, Welcome to Paisa" "What should we call you?" doesn't do anything. 
I already had this issue with the older version, but I managed to somehow work around it by deleting it's data and trying again.
The button lights up as if it was pressed, but doesn't do anything.

-----
Phone: Galaxy A33 5G
Android version: 14
One UI version: 6.0

**Comment by im-skylar** [comment]
While I'm having issues building the project I did get some logs from running it on my phone:
<details><summary>Log</summary>
<p>

```
Showing SM A336B logs:
I/flutter ( 4370): dynamic_color: Core palette detected.
I/flutter ( 4370): Provider<Box<dynamic>> not found for UserOnboardingPage
I/flutter ( 4370): #0      Provider._inheritedElementOf (package:provider/src/provider.dart:343)
I/flutter ( 4370): #1      Provider.of (package:provider/src/provider.dart:293)
I/flutter ( 4370): #2      _UserOnboardingPageState.saveName (package:paisa/features/intro/presentation/pages/user_onboarding_page.dart:76)
I/flutter ( 4370): #3      _UserOnboardingPageState.build.<anonymous closure> (package:paisa/features/intro/presentation/pages/user_onboarding_page.dart:122)
I/flutter ( 4370): #4      _InkResponseState.handleTap (package:flutter/src/material/ink_well.dart:1183)
I/flutter ( 4370): #5      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:275)
I/flutter ( 4370): #6      TapGestureRecognizer.handleTapUp (package:flutter/src/gestures/tap.dart:652)
I/flutter ( 4370): #7      BaseTapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:309)
I/flutter ( 4370): #8      BaseTapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:242)
I/flutter ( 4370): #9      PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:630)
I/flutter ( 4370): #10     PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:98)
I/flutter ( 4370): #11     PointerRouter._dispatchEventToRoutes.<anonymous closure> (package:flutter/src/gestures/pointer_router.dart:143)
I/flutter ( 4370): #12     _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:633)
I/flutter ( 4370): #13     PointerRouter._dispatchEventToRoutes (package:flutter/src/gestures/pointer_router.dart:141)
I/flutter ( 4370): #14     PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:127)
I/flutter ( 4370): #15     GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:488)
I/flutter ( 4370): #16     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:468)
I/flutter ( 4370): #17     RendererBinding.dispatchEvent (package:flutter/src/rendering/binding.dart:439)
I/flutter ( 4370): #18     GestureBinding._handlePointerEventImmediately (package:flutter/src/gestures/binding.dart:413)
I/flutter ( 4370): #19     GestureBinding.handlePointerEvent (package:flutter/src/gestures/binding.dart:376)
I/flutter ( 4370): #20     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:323)
I/flutter ( 4370): #21     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:292)
I/flutter ( 4370): #22     _invoke1 (dart:ui/hooks.dart:328)
I/flutter ( 4370): #23     PlatformDispatcher._dispatchPointerDataPacket (dart:ui/platform_dispatcher.dart:410)
I/flutter ( 4370): #24     _dispatchPointerDataPacket (dart:ui/hooks.dart:262)
```

</p>
</details>

**Comment by h4h13** [comment]
This is fixed in v6.0.3 can you verify
